src/common/drain/drain.js

```diff
@@ -42,9 +42,10 @@ function curateRegistry (agentIdentifier) {
  * @param {string} agentIdentifier - A unique 16 character ID corresponding to an instantiated agent.
  * @param {string} featureName - A named group into which the feature's buffered events are bucketed.
  */
-export function drain (agentIdentifier = '', featureName = 'feature') {
+export function drain (agentIdentifier = '', featureName = 'feature', force) {
   curateRegistry(agentIdentifier)
 
+  if (force) registry[agentIdentifier].delete(featureName)
```

Comment:
Do we ever use the `force` flag?

Reply:
all the features use it if the "auto" property is false